task10-2/task10-2/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task10_2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите количество студентов");
+            int m;
+
+            if (!TryInputNumber(out m) || m <= 0)
+            {
+                Console.ReadKey();
+                return;
+            }
+            
+            Console.WriteLine("Введите количество экзаменов");
+            int k;
+
+            if (!TryInputNumber(out k) || k <= 0)
+            {
+                Console.ReadKey();
+                return;
+            }
+
+            for (int i = 1; i <= m; i++)
+            {
+                int totalScore = 0;
+                Console.WriteLine($"Введите оценки студента {i} через пробел:");
```

Comment:
Сообщение вводит пользователя в заблуждение
```suggestion
                Console.WriteLine($"Введите оценки студента {i} через Enter:");
```